sdk/eventgrid/azure-eventgrid/samples/champion_scenarios/cs1b_publish_custom_events_to_a_topic_with_signature.py

```diff
@@ -1,8 +1,20 @@
+"""
+FILE: cs1b_publish_custom_events_to_a_topic_with_signature.py
+DESCRIPTION:
+    These samples demonstrate sending an EventGrid Event using a shared access signature for authentication.
+USAGE:
+    python cs1b_publish_custom_events_to_a_topic_with_signature.py
+    Set the environment variables with your own values before running the sample:
+    1) EG_ACCESS_KEY - The access key of your eventgrid account.
```

Comment:
we specify these for some but not for others?

Reply:
yeah - consumer samples don't have any env variables - these are file specific